packages/lit-virtualizer/src/lib/lit-virtualizer.ts

```diff
@@ -37,11 +39,23 @@ export class LitVirtualizer extends LitElement {
         }
     }
 
+    /**
+     * Scroll to the specified index, placing that item at the given position
+     * in the scroll view.
+     */
+    async scrollToIndex(index: number, position: string = 'start') {
+        this._scrollTo = {index, position}
+        this.requestUpdate()
+        await this.updateComplete
+        this._scrollTo = null
+    }
+
     render(): TemplateResult {
         return html`${scroll({
             items: this.items,
             template: this._template,
             scrollTarget: this.scrollTarget,
+            scrollTo: this._scrollTo,
```

Comment:
@graynorton Do you think the option on the `scroll` directive should also be named `scrollToIndex`? For consistency?

Reply:
Yeah, I think that would probably be best, now that you mention it. :-)